Clean up and organize (#15)
rsksmart/rif-identity-docs: ssi/applications/index.markdown

## Applications

> - [Issuer app](./issuer-app) - application that serves as the credential request manager. It allows to grant-deny requests or revoke existing credentials

Reviewer: ```suggestion
- [Issuer app](./issuer-app) - web application that serves as the credential request manager. It allows to grant-deny requests, issue verifiable credentials and revoke existing ones.
```

> - [Holder app](./holder-app) - wallet used to store declarative details and credentials of it’s users

Reviewer: ```suggestion
- [Holder app](./holder-app) - mobile wallet used to store verifiable credentials and declarative details of it’s users
```

> - [Verifier app](./verifier-app) - QR scanner app that verifies Verifiable Presentations

Reviewer: ```suggestion
- [Verifier app](./verifier-app) - QR scanner mobile app that verifies verifiable presentations (of verifiable credentials)
```
